htdocs/admin/ticket.php

```diff
@@ -219,7 +228,7 @@
 
 $dirmodels = array_merge(array('/'), (array) $conf->modules_parts['models']);
 
-$form = new Form($db);
+$form = new FormCategory($db);
```

Comment:
The var $form is commonly used for new Form($db)
This may create conflict. Can you also rename the var $form into $formcategory ?

Reply:
Sure !